src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -225,8 +231,8 @@ export class CalciteTipManager {
         >
           <CalciteIcon size="16" path={dir === "ltr" ? chevronLeft16 : chevronRight16} />
         </calcite-action>
-        <span class={CSS.pagePosition}>
-          {`${this.textPaginationLabel} ${this.selectedIndex + 1}/${this.total}`}
+        <span key={selectedIndex} class={CSS.pagePosition}>
```

Comment:
I'm wondering if needing this is a Stencil bug. `selectedIndex` should trigger rendering.

Reply:
Yeah, it does seem like it should update on render. Do you want to investigate?

Reply:
That's a Texas-sized 10-4.

Reply:
Issue: https://github.com/Esri/calcite-app-components/issues/327